### PR TITLE
readme.txt enhancements

### PR DESCRIPTION
**Pull request by luciano-croce**
- Minor enhancements to improve descriptions (for example: 4.9+ instead 4.9)
- Add tag: blocks
- Ordered the name of the contributors in this order: Author - Co-Author - Contributors (in alphabetical order)

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -37,7 +37,7 @@ Fixed a bug where it may attempt to load the Block Editor for post types that do
 Updated for WordPress 5.0.
 Changed all "Gutenberg" names/references to "Block Editor".
 Refreshed the settings UI.
-Removed disabling of the Gutenberg plugin. This was added for testing in WordPress 4.9. Users who want to continue following the development of Gutenberg in WordPress 5.0 and beyond will not need another plugin to disable it.
+Removed disabling of the Gutenberg plugin. This was added for testing in WordPress 4.9+ Users who want to continue following the development of Gutenberg in WordPress 5.0 and beyond will not need another plugin to disable it.
```
Comment (azaozz) [inline readme.txt]: `4.9+` is not that clear, thinking most people will understand `version 4.9` though :)

File: readme.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 === Classic Editor ===
-Contributors: azaozz, melchoyce, chanthaboune, alexislloyd, pento, youknowriad, desrosj, luciano-croce
-Tags: editor, classic editor, block editor, gutenberg
+Contributors: azaozz, luciano-croce, alexislloyd, chanthaboune, desrosj, melchoyce, pento, youknowriad
```
Comment (azaozz) [inline readme.txt]: The contributor names were added somewhat randomly, think that's good enough? :)

File: readme.txt (hunk 1 above)
Comment (luciano-croce) [reply readme.txt]: OK

File: readme.txt (hunk 2 above)
Comment (luciano-croce) [reply readme.txt]: OK

**Comment by azaozz** [comment]
Lets keep it as-is for now and look again when editing these places in the future :)

**Comment by luciano-croce** [comment]
> Lets keep it as-is for now and look again when editing these places in the future :)

OK sorry @azaozz 😄

**Pull request closed without merging**